lib/Froxlor/Api/Commands/Certificates.php

```diff
@@ -345,14 +348,16 @@ private function addOrUpdateCertificate($domainid = 0, $ssl_cert_file = '', $ssl
 			`ssl_cert_file` = :ssl_cert_file,
 			`ssl_key_file` = :ssl_key_file,
 			`ssl_ca_file` = :ssl_ca_file,
-			`ssl_cert_chainfile` = :ssl_cert_chainfile
+			`ssl_cert_chainfile` = :ssl_cert_chainfile,
+			`expirationdate` = :expirationdate
 			" . $qrywhere . " `domainid`= :domainid
 		");
 		$params = array(
 			"ssl_cert_file" => $ssl_cert_file,
 			"ssl_key_file" => $ssl_key_file,
 			"ssl_ca_file" => $ssl_ca_file,
 			"ssl_cert_chainfile" => $ssl_cert_chainfile,
+			"expirationdate" => date("Y-m-d H:i:s", $expirationdate),
```

Comment:
this will "fail" if `$expirationdate` is **null** - it then outputs `1970-01-01 01:00:00`

Reply:
Ops, you're right, I change the way the variable is assigned to query.